docs/reference/contrib/modeladmin/indexview.rst

```diff
@@ -25,16 +25,10 @@ on the ``ModelAdmin`` class itself.
 
 Default value: ``('__str__',)``
 
-Set ``list_display`` to control which fields are displayed in the IndexView
+Set ``list_display`` to control which fields are displayed in the ``IndexView``
 for your model.
 
-Example
-
-```
-list_display = ('first_name', 'last_name')
```

Comment:
At first I was going to fix formatting for this one, but I decided to remove it because the same line is used in the following example.